src/polymer/polymer-element.ts

```diff
@@ -262,9 +262,9 @@ function resolveElement(
     scannedElement: ScannedPolymerElement, document: Document): PolymerElement {
   const element = new PolymerElement();
   element.privacy = scannedElement.privacy;
-  applySuperClass(element, scannedElement, document);
-  applyMixins(element, scannedElement, document);
   applySelf(element, scannedElement, document);
+  applyMixins(element, scannedElement, document);
+  applySuperClass(element, scannedElement, document);
```

Comment:
Yeah, I think that's the best solution. I've got enough big changes swinging around right now though. I'll land these three and then take a look at converting them to maps. Tons of usages of these right now involve trying to awkwardly treat these arrays as maps.